src/style.css

```diff
@@ -112,6 +112,7 @@ body {
 .focus-search-area:focus {
   border: none;
   outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
the background should be added to the whole form, not just input
![image](https://user-images.githubusercontent.com/57598825/148947313-bb51360d-85fd-4aa4-907d-33854944fda6.png)

Reply:
Ok, I fixed it.

> the background should be added to the whole form, not just input ![image](https://user-images.githubusercontent.com/57598825/148947313-bb51360d-85fd-4aa4-907d-33854944fda6.png)